### PR TITLE
Feature/macos legacy

### PR DESCRIPTION
**Pull request by camilasan**
Trying to build mac OS legacy versions from master.

### CODE REVIEW

**Comment by allexzander** [comment]
lgtm, does it compile on the build machine? I would have @claucambra here anyways for an expert look

File: shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -710,7 +710,6 @@
 					"@executable_path/../Frameworks",
 					"@executable_path/../../../../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 11.0;
```
Comment (claucambra) [inline shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj:713]: This breaks our Xcode integration

Comment (claucambra) [reply shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj:713]: Same goes for the other removals of the deployment targets in the pbxproj

**Review by claucambra — CHANGES_REQUESTED** [review]
I think the pbxproj changes should be reverted or amended, but everything else looks great :)

File: shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj (hunk 1 above)
Comment (claucambra) [reply shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj:713]: I don't think we can, but then again I don't think we need to, this doesn't affect the build we run via CMake (the deployment target on the xcodebuild run in the shell integration CMakeFiles is set by CMake). This is mainly for the IDE integration. I think we should set 11.0 as the deployment target for the File Provider extension and whatever our intended minimum target is for the Finder Sync Extension (10.13 or 10.14)

**Comment by tobiasKaminsky** [comment]
Can we then backport this to 3.12?

**Review by tobiasKaminsky — APPROVED** [review]

**Comment by labor4** [comment]
Much love from me for this effort! Thanks

**Pull request merged**